concurrent/shared/src/main/scala/zio/concurrent/ConcurrentMap.scala

```diff
@@ -30,11 +30,11 @@ final class ConcurrentMap[K, V] private (private val underlying: ConcurrentHashM
     }
 
   /**
-   * Attempts to compute a mapping for the given key and its current mapped
-   * value.
+   * Attempts to compute a mapping for the specified key and its current mapped
+   * value (or None if there is no current mapping).
    */
-  def compute(key: K, remap: (K, V) => V): UIO[Option[V]] =
-    ZIO.succeed(Option(underlying.compute(key, remapWith(remap))))
+  def compute(key: K, remap: (K, Option[V]) => V): UIO[V] =
```

Comment:
I think the signature here should probable be `(K, Option[V]) => Option[B]` since the remapping function also has the ability to remove the entry.

Reply:
Oh, it also removes entry on null. Thanks! Fixed.

Probably here `(K, Option[V]) => Option[B]` `Option[B]` is a typo and I assume you meant `Option[V]`.

Reply:
Yes sorry! 😃